components/finder.rst

```diff
@@ -82,7 +82,7 @@ directory to use for the search::
 Search in several locations by chaining calls to
```

Comment:
Hmm, it's less readable...

Is it ok for you to suggest the 2 options ? See my update :)